PhysicsTools/NanoAOD/python/nano_cff.py

```diff
@@ -322,3 +322,29 @@ def nanoWmassGenCustomize(process):
                run2_nanoAOD_106Xv1 )
 _modifiers.toModify(linkedObjects,lowPtElectrons="")
 _modifiers.toModify(simpleCleanerTable,lowPtElectrons="")
+
+###add items missing in Run-2 UL samples needed for V10
+def nanoAOD_customizeV10(process):
```

Comment:
we should not use the customise function here but we want to use modifier to switch the mini, and we might need to nest this also in the V11, V12 ...


_modifiers = (run2_nanoAOD_106Xv2 | run3_nanoAOD_122)
(_modifiers) call your function to re-run the ID

Reply:
DeepTauID v2p5 will be added with `run2_nanoAOD_106Xv2` and `run3_nanoAOD_122` modifiers using existing function to add tauIDs; the customization function `nanoAOD_customizeV10` will be removed.